includes/class-profile.php

```diff
@@ -70,7 +151,7 @@ public static function newspack_get_option( $key ) {
 	 * @param string $value The option value.
 	 */
 	public static function newspack_update_option( $key, $value ) {
-		update_option( NEWSPACK_PROFILE_OPTION_PREFIX . $key, $value );
+		update_option( self::$opt_prefix . $key, $value );
```

Comment:
Am I crazy or is `$opt_prefix` not defined anywhere?

Reply:
Forgot to remove this method in ef196e588835342758b0ac3b17057b0faa7e7d08, fixed in 967f3b788d6558a421a5f182162719228d3b9dff